airflow/logging_backend.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow import configuration
+from airflow.logging_backends.base_logging_backend import BaseLoggingBackend
+from airflow.utils.imports import import_class_by_name
+
+LOGGING_BACKEND = None
```

Comment:
Let's encapsulate this file with a class, it should also have a descriptive top-level comment, since there is that confusion we discussed of what is used to write to logs vs reading them.